test/CMakeLists.txt

```diff
@@ -61,6 +61,24 @@ add_test(NAME LCFIJet
 )
 set_property(TEST LCFIJet APPEND PROPERTY DEPENDS ddsim_edm4hep)
 
+add_test(NAME tracking_truth
+         WORKING_DIRECTORY ${CLDConfig_DIR}
+         COMMAND k4run --trackingOnly --truthTracking --inputFiles=test.edm4hep.root --outputBasename=trkOnly_test_edm4hep CLDReconstruction.py --GeoSvc.detectors=${DETECTOR}
+)
+set_property(TEST tracking_truth APPEND PROPERTY DEPENDS ddsim_edm4hep)
+
+add_test(NAME tracking_conformal
+         WORKING_DIRECTORY ${CLDConfig_DIR}
+         COMMAND k4run --trackingOnly --conformalTracking --inputFiles=test.edm4hep.root --outputBasename=trkOnly_test_edm4hep CLDReconstruction.py --GeoSvc.detectors=${DETECTOR}
+)
+set_property(TEST tracking_conformal APPEND PROPERTY DEPENDS ddsim_edm4hep)
+
+add_test(NAME tracking_both
+         WORKING_DIRECTORY ${CLDConfig_DIR}
+         COMMAND k4run --trackingOnly --conformalTracking --truthTracking --inputFiles=test.edm4hep.root --outputBasename=trkOnly_test_edm4hep CLDReconstruction.py --GeoSvc.detectors=${DETECTOR}
```

Comment:
```suggestion
         COMMAND k4run --trackingOnly --conformalTracking --truthTracking --inputFiles=test.edm4hep.root --outputBasename=trkOnly_both_test_edm4hep CLDReconstruction.py --GeoSvc.detectors=${DETECTOR}
```

Reply:
this is not running anything anyway so I removed all the leftover arguments